### DISCUSSION TITLE
Button -> Nozzle Cleaning/Maintance

### DISCUSSION THREAD

**Discussion by tharighteous** [OrcaSlicer/OrcaSlicer#2381] [Dev]
Hi.
is it pssible to add a "Maintanance Button" in order to clean the nozzle.
So if its dirty/clogged the hotend parks at the "Filament basket" of my bambu x1c.

you have to pause and use the hotend arrow keys in the software or on the panel an so on.
to complicated.

thanks

**Comment by foreachthing** [comment]
You could write your own "clean nozzle" gcode and save it. That way you can just send/exec this file to do it's job.
